src/components/SearchResultsCategory.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+
+export default function SearchResultsCategory(): JSX.Element {
+  const classes = useStyles();
+  return (
+    <div className={classes.paper}>
+      <Typography component="h2" variant="h6">
```

Comment:
I see this a lot in Material-UI examples. Do you know why they do it? Why bother changing the component to an 'h2' but display an 'h6' variant?

Reply:
I know that it is actually the variant that applies the type of style, but for component I thinks it's in regards to the semantic element:
https://material-ui.com/components/typography/#changing-the-semantic-element
This also confuses me, 
'The Typography component uses the variantMapping property to associate a UI variant with a semantic element. _It’s important to realize that the style of a typography is independent from the semantic underlying element._
You can change the underlying element for a one time occasion with the component property'

Reply:
I think "semantic element" refers to how indexers, screen readers, etc., would understand the page. Maybe that's what they're doing -- using 'h6' styling but making it appear more important to computers trying to understand the page.